giza/agents/integration.py

```diff
@@ -0,0 +1,13 @@
+from ape.api import AccountAPI
+
+import giza.agents.integrations.uniswap.uniswap as uniswap_module
```

Comment:
Create an `__init__.py` inside integrations which imports `Uniswap` class.

`__init__.py` 
```python
from import giza.agents.integrations.uniswap.uniswap import Uniswap

__all__ = ["Uniswap"]
```

New import:

```python
from giza.agents.integrations import Uniswap
```

Another suggestion could be to call all the integration where the main integration is with a common name like `core.py` or `integration_handler.py` so all the new integrations could be referenced the same way

```python
from giza.agents.integrations.uniswap.core import UniswapV3
from giza.agents.integrations.enzyme.core import Enzyme
```